internal/bminventory/inventory.go

```diff
@@ -1952,6 +1946,34 @@ func (b *bareMetalInventory) validateUpdateCluster(
 	return params, nil
 }
 
+func getBoolPtrForLog(b *bool) string {
+	if b == nil {
+		return "nil"
+	}
+	return fmt.Sprintf("%t", *b)
+}
```

Comment:
maybe it would make sense to move it in the `internal/common` package? That way it would be easy to reuse it other places in the future?